src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs

```diff
@@ -80,11 +81,13 @@ public static class ServiceStackRedisIntegration
                 throw;
             }
 
+            RedisNativeClientData clientData = redisNativeClient.As<RedisNativeClientData>();
+
             using (var scope = RedisHelper.CreateScope(
                 Tracer.Instance,
                 IntegrationName,
-                GetHost(redisNativeClient),
-                GetPort(redisNativeClient),
+                clientData.Host,
```

Comment:
I don't know if `Host` can ever be null, but the original code coalesced it to an empty string if null.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/985/commits/bf1cab2035da1c82eac4b7f68784feb8ad477565 :)